extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -347,6 +347,10 @@ export class AmpAnalytics extends AMP.BaseElement {
 
   /** @private */
   getInlineConfigNoInline() {
+    if (this.element.CONFIG) {
```

Comment:
Could you please also put `@return` jsdoc above? Looks like it was omitted originally.